packages/@okta/courage-for-signin-widget/src/CourageForSigninWidget.ts

```diff
@@ -34,10 +34,20 @@ import TextBox from './courage/views/forms/inputs/TextBox';
 import Callout from './courage/views/components/Callout';
 import Backbone from 'backbone';
 
-import FrameworkView from './courage/framework/View';
-
+import FrameworkView, { FrameworkViewClass } from './courage/framework/View';
 import './util/scrollParent';
 
+const Controller = BaseController.extend({
+  // The courage BaseController renders asynchronously in current versions of jQuery
+  // https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/BaseController.js#L108
```

Comment:
the link has been changed
https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/BaseController.ts#L117-L119